backend/open_webui/main.py

```
@@ -778,128 +1567,445 @@ async def inspect_websocket(request: Request, call_next):
+            if metadata.get("chat_id") and metadata.get("message_id"):
+                try:
+                    if not metadata["chat_id"].startswith("local:"):
+                        Chats.upsert_message_to_chat_by_id_and_message_id(
+                            metadata["chat_id"],
+                            metadata["message_id"],
+                            {
+                                "parentId": metadata.get("parent_message_id", None),
+                                "model": model_id,
+                            },
+                        )
+                except:
```

Ruff (0.15.1)

[error] Do not use bare `except`

(E722)